@codex please fix

```
 building client (vite) 
 ✓ tests/navigateWithRetry.test.ts (2 tests) 10ms
 ✓ tests/k8sEnvironmentOverlays.test.ts (2 tests) 5ms
 ✓ tests/noConflictMarkers.test.ts (1 test) 1248ms
   ✓ repository sanity ‑ no merge-conflict markers > contains no <<<<<<< or >>>>>>> markers  1246ms
 ✓ scripts/tests/questDependencies.test.ts (2 tests) 6ms
 ✓ tests/svelteResolution.test.ts (4 tests) 522ms
 ✓ tests/architectureNotesAccessibilityBaseline.test.ts (2 tests) 5ms
 ✓ tests/questDependencyReferences.test.ts (1 test) 42ms
 ✓ tests/questIdPathConsistency.test.ts (1 test) 35ms
 ✓ tests/svelteSubpathResolver.test.ts (2 tests) 7ms
 ✓ tests/serviceWorkerConfigCache.test.ts (1 test) 5ms
 ✓ tests/noDuplicateItemIds.test.ts (1 test) 7ms
 ✓ frontend/tests/docsMarkdownSafety.test.ts (1 test) 14ms
 ✓ tests/questReliabilityCoverage.test.ts (1 test) 196ms
 ✓ tests/v3ReleaseStateDoc.test.ts (3 tests) 6ms
 ✓ frontend/tests/questListClassifier.test.ts (3 tests) 7ms
 ✓ tests/changelogQuestBranchingNote.test.ts (1 test) 7ms
 ✓ tests/metricsEndpoint.test.ts (3 tests) 22ms
 ✓ tests/docsSearchFeatures.test.ts (2 tests) 8ms
 ✓ tests/questProcessRecoveryPaths.test.ts (1 test) 162ms
 ✓ tests/questsPolishPromptSections.test.ts (3 tests) 7ms
 ✓ tests/builtinQuestSchema.test.ts (1 test) 116ms
 ✓ tests/githubTokenPersistence.test.ts (1 test) 13ms
 ✓ tests/spellcheckConfig.test.ts (1 test) 5ms
 ✓ frontend/src/components/__tests__/QuestLinkChips.spec.ts (2 tests) 221ms
03:31:38 [build] Rearranging server assets...
03:31:38 [build] Server built in 35.83s
03:31:38 [build] Complete!
 ✓ frontend/__tests__/ShoppingForm.test.js (1 test) 4ms
 ✓ tests/menu-stats.test.ts (1 test) 6ms
 ✓ tests/profileTitles.test.ts (1 test) 5ms
 ✓ frontend/tests/menu.test.ts (1 test) 4ms
 ✓ frontend/tests/duration.test.ts (2 tests) 3ms
 ✓ tests/menuTitles.test.ts (1 test) 6ms
 ✓ tests/pnpmfile.test.ts (1 test) 6ms
 ✓ tests/domExceptionGlobal.test.ts (1 test) 6ms
 ✓ tests/domexception-global.test.ts (1 test) 5ms
 ✓ tests/domexception-globals.test.ts (1 test) 5ms
 ✓ tests/sysadminQuest.test.ts (1 test) 4ms
 ✓ tests/shopIndex.test.ts (1 test) 5ms
 ✓ tests/eslintPluginPresence.test.ts (1 test) 4ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 7ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 8ms
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 28226ms
   × Prettier formatting > frontend sources are formatted 28223ms
     → npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] src/utils/customcontent.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 5ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 4ms
 ✓ tests/prismSvelte.test.ts (1 test) 16ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 5ms
 ✓ tests/progressionBalance.test.ts (1 test) 51ms
 ✓ tests/domException.test.ts (1 test) 4ms
 ✓ frontend/tests/sanity.test.ts (1 test) 4ms
 ✓ tests/indexeddb.server.test.ts (1 test) 201ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] src/utils/customcontent.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13
     15|       const stdout = error?.stdout?.toString?.() ?? '';
     16|       const stderr = error?.stderr?.toString?.() ?? '';
     17|       throw new Error(`npm run format:check failed.\n${stdout}${stderr…
       |             ^
     18|     }
     19|   }, 240000);

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 292 passed (293)
      Tests  1 failed | 1041 passed (1042)
   Start at  03:29:48
   Duration  166.36s (transform 10.06s, setup 11.49s, collect 47.74s, tests 122.02s, environment 194.63s, prepare 37.72s)


Error: Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] src/utils/customcontent.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/src/utils/customcontent.js
+++ b/frontend/src/utils/customcontent.js
@@ -505,14 +505,17 @@ export async function importCustomContentString(dataString) {
         throw new Error('Invalid backup data');
     }
 
-    const isValidEntity = (entity) => entity && typeof entity === 'object' && !Array.isArray(entity);
+    const isValidEntity = (entity) =>
+        entity && typeof entity === 'object' && !Array.isArray(entity);
 
     if (![...items, ...processes, ...quests].every(isValidEntity)) {
         throw new Error('Invalid backup data');
     }
 
     const cloneEntity = (entity) =>
-        typeof structuredClone === 'function' ? structuredClone(entity) : JSON.parse(JSON.stringify(entity));
+        typeof structuredClone === 'function'
+            ? structuredClone(entity)
+            : JSON.parse(JSON.stringify(entity));
 
     await Promise.all([
         ...items.map((item) => db.items.add(cloneEntity(item))),

